doc/developers/bug_triaging.rst

```diff
@@ -148,5 +150,21 @@ The following workflow [*]_ is a good way to approach issue triaging:
    An additional useful step can be to tag the corresponding module e.g.
    `sklearn.linear_models` when relevant.
 
-.. [*] Adapted from the pandas project `maintainers guide
+Becoming a member of the scikit-learn triage team
+--------------------------------------------------
+
+Any contributor may become a member of the scikit-learn triage team, after
+showing some continuity in participating to scikit-learn
+development (with pull requests and reviews).
+Any core developer are welcome to propose a scikit-learn contributor to the
+triage team [2]_. Without objections from other core developers, the
+contributor will be added to the triage team.
```

Comment:
No objections feels more strict than becoming a core developer. Should we use the same voting policy we have for core developers?

> While it is expected that most votes will be unanimous, a two-thirds majority of the cast votes is enough.

Reply:
Shouldn't  this part (about the voting process) be in the governance doc instead of here?

Reply:
I was describing more a loose check for objections than a voting process.
If this implies a voting process, indeed, I will put it in the governance section.

Reply:
Maybe we should just start the sentence with "Similarly to the <python developers guide (include link)>, ..." and not have a footnote then

Reply:
or change the footnote to somethign like "this process is similar to..."